modules/editorial-metadata/editorial-metadata.php

```diff
@@ -1717,6 +1569,43 @@ public function print_configure_view()
                 <?php
             endif;
         }
+
+        /**
+         * Load the chain of input-type handlers.
+         *
+         * @since   @todo
+         */
+        public function load_input_handlers()
+        {
+            $handlers_base_path = dirname(__FILE__) . '/input-handlers';
+
+            require_once "{$handlers_base_path}/editorial-metadata-input-text-handler.php";
```

Comment:
Agreed that would be a nice and needed enhancement, but the `require_once`s were added since almost no modules seemed to be using namespaces and autoload features. Perhaps in another enhancement PR with the change?

Reply:
👍